classes/fields/number.php

```diff
@@ -263,10 +269,14 @@ public function validate( $value, $name = null, $options = null, $fields = null,
 		$label = pods_v( 'label', $options, ucwords( str_replace( '_', ' ', $name ) ) );
 
 		if ( 0 < strlen( $check ) ) {
-			return sprintf( __( '%s is not numeric', 'pods' ), $label );
+			$errors[] = sprintf( __( '%s is not numeric', 'pods' ), $label );
```

Comment:
:no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: d7b3f08af1cfdf3755a5d6925c86c8c6</sub>